README.md

```diff
@@ -1,6 +1,6 @@
-![Coverage](https://img.shields.io/badge/coverage-93%25-green)
+![Coverage](https://img.shields.io/codecov/c/github/openwallet-foundation-labs/sd-jwt-js)
 ![License](https://img.shields.io/github/license/openwallet-foundation-labs/sd-jwt-js.svg)
-![NPM](https://img.shields.io/npm/v/%40hopae%2Fsd-jwt)
+![NPM](https://img.shields.io/npm/v/%40sd-jwt%2Fcore)
```

Comment:
we should keep in mind that we have multiple packages so theoretically it's wrong to just point to one version.

One the other hand we keep all the versions in sync. Do you think we should mention this somewhere?

Reply:
To make it simple I suggest to sync the versions. Keeping them independant will increase the trouble on our side.

Maybe this PR is a good change tp add a reference in the contribution section.